File: Patterns/Manuscript/manuscript/code/java/DesignPatterns/src/main/java/abstractfactory/FantasyLandFactory.java
package abstractfactory;

public class FantasyLandFactory implements AbstractFoodStallFactory {
    public IceCream createIceCream() {

        System.out.println("Creating magical-themed ice cream");
        return new MagicalIceCream();
    }

    public Pizza createPizza() {
        System.out.println("Creating enchanted-themed pizza");
        return new EnchantedPizza();
    }
}

File: Patterns/Manuscript/manuscript/code/python/abstractfactory/FantasyLandFactory.py
from MagicalIceCream import MagicalIceCream
from EnchantedPizza import EnchantedPizza
from AbstractFoodStallFactory import AbstractFoodStallFactory


class FantasyLandFactory(AbstractFoodStallFactory):
    def createIceCream(self):
        print("Creating magical-themed ice cream")
        return MagicalIceCream()

    def createPizza(self):
        print("Creating enchanted-themed pizza")
        return EnchantedPizza()
